### PR TITLE
dev-libs/gjs: Version bump to 1.75.90 BETA

### PR DESCRIPTION
**Pull request by flint2**
Need mask, in preparation for Gnome 44

### CODE REVIEW

**Comment by flint2** [comment]
Error have_gtk4

**Comment by flint2** [comment]
> Error have_gtk4

Adding patch for meson.build

File: dev-libs/gjs/gjs-1.75.90.ebuild [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+LICENSE="MIT || ( MPL-1.1 LGPL-2+ GPL-2+ )"
+SLOT="0"
+IUSE="+cairo examples readline sysprof test"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~riscv ~x86"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=dev-libs/glib-2.66.0:2
+	dev-libs/libffi:=
+	>=dev-libs/gobject-introspection-1.66.1:=
```
Comment (mattst88) [inline dev-libs/gjs/gjs-1.75.90.ebuild:20]:
````diff
-if gi.version().version_compare('<1.66.1')
+if gi.version().version_compare('<1.71.0')
````

We should raise the required version.

Comment (flint2) [reply dev-libs/gjs/gjs-1.75.90.ebuild:20]: Ok, also changed 
-       >=dev-libs/gobject-introspection-1.66.1:=
+       >=dev-libs/gobject-introspection-1.71.1:=

**Comment by flint2** [comment]
Removed mask, will be included in separate PR

**Pull request closed without merging**